### PR TITLE
gitignore eggs, add loader function

### PR DESCRIPTION
**Pull request by jtclaypool**
please review

### CODE REVIEW

File: massholes/loader.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+
+def loader(path):
+    df = pd.read_excel(path,index_col=0)
+    treatment=pd.read_excel(path,sheet_name='y')
+    treatment=pd.Series(treatment['y'])
+
+    new_df=df.transpose().reset_index()
+    new_df['treatment']=treatment
```
Comment (JCZuurmond) [inline massholes/loader.py]: Can you use assign?

File: massholes/loader.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+
+def loader(path):
+    df = pd.read_excel(path,index_col=0)
+    treatment=pd.read_excel(path,sheet_name='y')
+    treatment=pd.Series(treatment['y'])
```
Comment (JCZuurmond) [inline massholes/loader.py]: I feel this line is redundant, simply user `treatment['y']` below. A column is already a `pd.Series`.

File: massholes/loader.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+
+def loader(path):
```
Comment (JCZuurmond) [inline massholes/loader.py]: A little documentation would be nice!

Comment (jtclaypool) [reply massholes/loader.py]: Added a single line docstring

**Pull request merged**